workflows/gtfs_validator_execution.yml

```diff
@@ -0,0 +1,187 @@
+main:
+  params: [event]
+  steps:
+      - log_event:
+          call: sys.log
+          args:
+              text: ${event}
+              severity: INFO
+      - extract_data:
+          assign:
+          - data: ${event.data.protoPayload}
+      - init:
+          assign:
+            - datasetsBucket: ${sys.get_env("datasets_bucket_name")}
+            - reportsBucket: ${sys.get_env("reports_bucket_name")}
+            - validatorEndpoint: ${sys.get_env("validator_endpoint")}
+            - feedID: ${text.split(data.resourceName, "/")[5]}
```

Comment:
Would it be possible to put a comment with a real example of a resourceName? So we can understand why [5] and [6] are used.